src/cmds/fs/more.c

```diff
@@ -28,29 +28,37 @@ static void screen(FILE *fp) {
 		columns = MAX_SCREEN_WIDTH - 1;
 	}
 	lines = std->endy - std->begy;
-	endwin();
+	//endwin();
```

Comment:
If this code is unused now you can safely delete it (that's another reason why we use VCS afterall).